polaris-react/src/components/Checkbox/Checkbox.scss

```diff
@@ -141,15 +112,10 @@
   display: block;
   width: 100%;
   height: 100%;
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  @include focus-ring($border-width: var(--p-border-width-2));
 
-  #{$se23} & {
-    /* stylelint-disable-next-line polaris/border/polaris/at-rule-disallowed-list -- se23 override */
-    @include no-focus-ring;
-    border: var(--p-border-width-1-experimental) solid
-      var(--p-color-border-input);
-  }
+  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+  @include no-focus-ring;
```

Comment:
```suggestion
```

Pretty sure this was here just to undo the above `focus-ring`, so can be deleted now.